### ISSUE TITLE
[MU4 Issue] Last note repeated on pause

### ISSUE THREAD

**Issue by moving-bits** [musescore/MuseScore#14141]
**Describe the bug**
MU sometimes plays an extra note shortly after begin of pause (repeating the last note shortly), sounds like sort of a "hiccup", before continuing as noted

**To Reproduce**
- I imported a score from MS3 and set instruments (choir voices) to MuseSound
- Play the score
- It does not happen for the first pauses, only after app. 20 measures or so (in another score it happens after app. 40 measures)

**Expected behavior**
Quiet during a pause

**Platform information**
 - Win 10 64bit, AMD, Realtek HD Audio
 - MuseScore 4 Beta

**Additional context**
- one score is SAB choir only, the other one I've tried SATB choir + Grand Piano
- both scores work flawlessly in MS3

Anything I can try or document?

**Comment by MarcSabatella** [comment]
This is a duplicate of https://github.com/musescore/MuseScore/issues/14003, which is closed as it was at least partially fixed already.  But at least in my test of the update that was pushed yesterday, the fix was only for sopranos, not the other choir sounds.  See also https://github.com/musescore/MuseScore/issues/14076

**Comment by moving-bits** [comment]
For me it's been bassos. I can provide the MS3 file (or the converted MS4 version) if it helps for debugging.

**Comment by melis-sa** [comment]
Only with the musesound Choir : in the last measure a 200% fermata on a quarter note followed by a quarter rest produces an echo, checked in Audacity. This does not happen with MS Basic sounds or MS3. Even with the latest nightly (28/10) all parts are affected. 
Audacity end MS Sound Choir
![Audacity end of MS4 score with  MSounds](https://user-images.githubusercontent.com/116520510/198746926-71e224f6-2b4c-403a-9123-d37148a1a066.png)
Audacity end MSBasic sounds
![Audacity end of MS4 score with MS basic](https://user-images.githubusercontent.com/116520510/198746924-a92ad372-0e3b-440e-be4b-f5b06e3fedd6.png)

**Comment by eg0n1984** [comment]
I have detected this in Choir piece playback with a 1/8 pause
https://www.dropbox.com/s/za5oi6ea864ogim/Gloriosa_Mae_de_Deus_MCarneiro.mp3?dl=0
![imagem](https://user-images.githubusercontent.com/28876087/198823410-d68abd47-2568-4c5d-b0c7-a9b34e4d3616.png)

**Comment by MarcSabatella** [comment]
Worth testing again with the latest update to Muse Sounds.  For me, the issue is now fixed.

**Comment by moving-bits** [comment]
Thanks for notifying us. How can I check whether MuseHub has updated the sounds? MuseHub is set to auto-update sounds, but on playing the score it still has the described effects.
Is there a way to manually update the choir sound files other the uninstall / reinstall?

**Comment by MarcSabatella** [comment]
Best to attach your score (or a link to it) and give precise steps to reproduce the problem

**Comment by moving-bits** [comment]
It's copyrighted (but licensed) material I've entered into MS for our choir. Is there a way to send this via PM?

**Comment by MarcSabatella** [comment]
There won't be an issue posting it to a bug tracker.  In the *extraordinarily* unlikely event the copyright owner ever finds out, they'll simply ask it be deleted; you won't be in trouble at all.

But, to be even safer, you could delete all but the relevant measures.

**Comment by moving-bits** [comment]
Ok, here is a pretty much reduced version for which I was able to reproduce the problem:

[Sample.zip](https://github.com/musescore/MuseScore/files/9905554/Sample.zip)

To reproduce:
- rename file to `.mscz` and open it in MS4
- open mixer, set Soprano to Solo, play
- have a look at the rest in measure 5 and the one at the beginning of measure 10

**Comment by Tantacrul** [comment]
Duplicate of https://github.com/musescore/MuseScore/issues/14003

Closing for this reason, although please do tell us if the issue is resolved for you.

**Comment by Tantacrul** [comment]
OK, I've tested the file and confirmed the issue that appears to remain for choir. To save headaches, I've cleared up the conversation and put the relevant info in the issue description.
